eos-updater-avahi/eos-updater-avahi.c

```diff
@@ -222,7 +227,7 @@ update_service_file (gboolean       advertise_updates,
 
   if (commit_checksum != NULL)
     {
-      commit_date_time = g_date_time_new_from_unix_utc (commit_timestamp);
+      commit_date_time = g_date_time_new_from_unix_utc ((gint64) commit_timestamp);
```

Comment:
Unfortunate that `g_date_time_new_from_unix_utc` takes `gint64` here instead of `guint64`. Should we do a range check to make sure that we don't end up with a negative value and instead wrap around?

Reply:
I’m not too fussed about the difference between `guint64` and `gint64` for timestamps. We’re going to hit the year 2038 problem for `gint32` in 20 years; the problems with 64-bit integers are waaaaaay off.

Reply:
Yeah, fair enough